### ISSUE TITLE
[Bug]: Helmet loss

### ISSUE THREAD

**Issue by cawieand91** [Antistasi-Ultimate-Community/A3-Antistasi-Ultimate#321]
### Describe the bug

Helmet loss occurs despite game setting being set to never.

### How to reproduce

occurs during gameplay with helmet loss change set to never

### Version

11.1.0

### Have you altered the code?

No

### Map

Altis, New Esseker

### What server?

Private dedicated server

### Mods

```html
<?xml version="1.0" encoding="utf-8"?>
<html>
  <!--Created by Arma 3 Launcher: https://arma3.com-->
  <head>
    <meta name="arma:Type" content="preset" />
    <meta name="arma:PresetName" content="Server Plus" />
    <meta name="generator" content="Arma 3 Launcher - https://arma3.com" />
    <title>Arma 3</title>
    <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet" type="text/css" />
    <style>
body {
	margin: 0;
	padding: 0;
	color: #fff;
	background: #000;	
}

body, th, td {
	font: 95%/1.3 Roboto, Segoe UI, Tahoma, Arial, Helvetica, sans-serif;
}

td {
    padding: 3px 30px 3px 0;
}

h1 {
    padding: 20px 20px 0 20px;
    color: white;
    font-weight: 200;
    font-family: segoe ui;
    font-size: 3em;
    margin: 0;
}

em {
    font-variant: italic;
    color:silver;
}

.before-list {
    padding: 5px 20px 10px 20px;
}

.mod-list {
    background: #222222;
    padding: 20px;
}

.dlc-list {
    background: #222222;
    padding: 20px;
}

.footer {
    padding: 20px;
    color:gray;
}

.whups {
    color:gray;
}

a {
    color: #D18F21;
    text-decoration: underline;
}

a:hover {
    color:#F1AF41;
    text-decoration: none;
}

.from-steam {
    color: #449EBD;
}
.from-local {
    color: gray;
}

</style>
  </head>
  <body>
    <h1>Arma 3  - Preset <strong>Server Plus</strong></h1>
    <p class="before-list">
      <em>To import this preset, drag this file onto the Launcher window. Or click the MODS tab, then PRESET in the top right, then IMPORT at the bottom, and finally select this file.</em>
    </p>
    <div class="mod-list">
      <table>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">CBA_A3</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=450814997" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=450814997</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">RKSL Studios - Attachments v3.02</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=1661066023" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=1661066023</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">RHSAFRF</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=843425103" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=843425103</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">RHSUSAF</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=843577117" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=843577117</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">CUP Weapons</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=497660133" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=497660133</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">CUP Terrains - Core</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=583496184" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=583496184</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">CUP Terrains - Maps</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=583544987" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=583544987</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">Interiors for CUP</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=1883956552" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=1883956552</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">Low Grass CUP Terrains</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=1350216151" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=1350216151</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">CUP Terrains - CWA</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=853743366" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=853743366</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">CUP Terrains - Maps 2.0</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=1981964169" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=1981964169</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">3CB BAF Equipment</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=893328083" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=893328083</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">3CB BAF Vehicles</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=893349825" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=893349825</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">3CB BAF Weapons</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=893339590" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=893339590</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">ace</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=463939057" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=463939057</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">CUP Units</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=497661914" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=497661914</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">Remove stamina</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=632435682" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=632435682</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">RHSGREF</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=843593391" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=843593391</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">Zeus Enhanced</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=1779063631" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=1779063631</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">VSM All-In-One Collection</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=823636749" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=823636749</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">RHSSAF</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=843632231" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=843632231</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">SPS AI AXMC Sniper Rifle Series</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=1510335080" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=1510335080</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">3CB BAF Units</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=893346105" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=893346105</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">ACE3 Arsenal Extended - Core</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=2522638637" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=2522638637</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">Tier One Weapons</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=2268351256" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=2268351256</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">Virolahti - Valtatie 7</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=1926513010" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=1926513010</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">Anizay</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=1537973181" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=1537973181</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">Kujari</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=1726494027" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=1726494027</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">CUP Vehicles</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=541888371" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=541888371</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">Ai accuracy fix</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=619834340" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=619834340</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">Recoil coefficient changer</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=1782747712" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=1782747712</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">Loot to Vehicle for ACE and Antistasi</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=1439779114" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=1439779114</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">Remove stamina - ACE 3</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=782415569" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=782415569</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">Advanced Weapon Mounting</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=1378046829" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=1378046829</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">BackpackOnChest</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=820924072" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=820924072</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">Zeus Enhanced - ACE3 Compatibility</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=2018593688" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=2018593688</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">ACSTG AI Cannot See Through Grass</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=2946868556" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=2946868556</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">VSM - ACE3 Compatibility</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=1974559189" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=1974559189</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">MK 13 Secondary GL</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=2975055854" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=2975055854</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">KT sniper rife</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=2464731269" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=2464731269</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">ACE3 Compatibility for SPS AI AXMC Sniper Rifles</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=1511234414" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=1511234414</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">Advanced Zipline</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=1569116504" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=1569116504</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">WebKnight's Two Primary Weapons</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=2976569274" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=2976569274</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">DUI - Squad Radar</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=1638341685" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=1638341685</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">Enhanced Map Ace Version</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=2467590475" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=2467590475</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">3CB BAF Vehicles (Servicing extension)</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=1135543967" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=1135543967</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">3CB BAF Units (RHS compatibility)</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=1135541175" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=1135541175</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">3CB BAF Units (ACE compatibility)</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=1135539579" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=1135539579</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">3CB BAF Weapons (RHS ammo compatibility)</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=1515845502" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=1515845502</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">3CB BAF Vehicles (RHS reskins)</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=1515851169" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=1515851169</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">3CB Factions</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=1673456286" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=1673456286</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">NGA - Next Generation Armory</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=3100490558" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=3100490558</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">Reduced Haze Mod v3.1</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=1397683809" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=1397683809</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">GC's No Recoil, No Weapon Sway</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=833939663" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=833939663</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">GC's Full Screen Night Vision</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=833942999" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=833942999</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">GC's Low Grass</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=834220345" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=834220345</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">No Weapon Sway</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=570118882" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=570118882</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">WMO - Walkable Moving Objects</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=925018569" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=925018569</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">Advanced Urban Rappelling</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=730310357" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=730310357</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">Advanced Sling Loading</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=615007497" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=615007497</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">Advanced Rappelling</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=713709341" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=713709341</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">Advanced Towing</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=639837898" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=639837898</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">Enhanced Movement</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=333310405" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=333310405</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">Pook ARTY Pack</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=1312457117" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=1312457117</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">RHSPKL</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=1978754337" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=1978754337</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">Inventory Weight Limit Increase</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=2177846178" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=2177846178</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">MapTools Remastered</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=3032131959" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=3032131959</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">Simple MapTools</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=2013446344" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=2013446344</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">Kunduz Afghanistan - Fixed Doors</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=1188303655" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=1188303655</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">Tembelan Island</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=1252091296" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=1252091296</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">UMB Colombia</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=2266710560" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=2266710560</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">Pulau</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=1423583812" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=1423583812</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">Antistasi Ultimate - Mod</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=3020755032" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=3020755032</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">Esseker</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=498101407" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=498101407</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">G.O.S Al Rayak</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=648172507" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=648172507</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">Namalsk [R]</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=3013966707" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=3013966707</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">New Esseker</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=2712204589" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=2712204589</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">Ruha</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=1368857262" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=1368857262</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">Saint Kapaulio</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=939686262" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=939686262</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">Šumava</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=2947655994" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=2947655994</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">Isla Abramia</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=648775794" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=648775794</a>
          </td>
        </tr>
      </table>
    </div>
    <div class="dlc-list">
      <table />
    </div>
    <div class="footer">
      <span>Created by Arma 3 Launcher by Bohemia Interactive.</span>
    </div>
  </body>
</html>
```

**Comment by Silence-112** [comment]
Already fixed in #310

**Comment by Silence-112** [comment]
Currently in [Public Testing](https://steamcommunity.com/sharedfiles/filedetails/?id=3169463443)
